<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SuperTest</class>
 <widget class="QMainWindow" name="SuperTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QPushButton" name="pushButton_load_test_dir">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string>加载单元测试目录</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_open_test_dir">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string>打开目录</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_reload_dir">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string>重新加载</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_load_path">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>441</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>63</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QListWidget" name="listWidget_load_dir">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTextEdit" name="textEdit_test_content">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>192</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>显示测试内容</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_test_result">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>192</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>显示测试结果内容</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>985</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_scan_test_dir"/>
    <addaction name="action_new_ut_instance"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_scan_test_dir">
   <property name="text">
    <string>扫描测试目录</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_new_ut_instance">
   <property name="text">
    <string>创建单元测试</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
